CHANGELOG.md

```diff
@@ -1,5 +1,71 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.2.0
+
+<!-- generated comparing v5.1.1..master -->
+
+_Nov 22, 2021_
+
+A big thanks to the 13 contributors who made this release possible. Here are some highlights ✨:
+
+- 🧪 Created another unstyled component: TabsUnstyled (#29597) @mnajdova.
+- 🎉 Updated the Material Icons set with the latest changes from Google (#29328) @michaldudak.
+- 🐛 Fixed bugs and improved the infrastructure and documentation 📚.
+
+### `@mui/material@5.2.0`
+
+- &#8203;<!-- 21 -->[i18n] Add the amharic language (#29153) @NatiG100
+- &#8203;<!-- 18 -->[material] Fix types for `variants.style` to accept callbacks (#29610) @mnajdova
+- &#8203;<!-- 16 -->[Popper] Simplify prop types (#29680) @michaldudak
+- &#8203;<!-- 15 -->[Select] Include aria-selected=false when option not selected (#29695) @michaldudak
+- &#8203;<!-- 06 -->[useMediaQuery] Fix crash in Safari < 14 and IE 11 (#29776) @eps1lon
+- &#8203;<!-- 05 -->[useMediaQuery] Ensure no tearing in React 18 (#28491) @eps1lon
+
+### `@mui/icons-material@5.2.0`
+
+- &#8203;<!-- 20 -->[icons] Sync recent Material Icons from Google (#29328) @michaldudak
+
+### `@mui/system@5.2.0`
+
+- &#8203;<!-- 13 -->[system] Fix minor CssVars issues (#29747) @siriwatknp
+
+### `@mui/styled-engine@5.2.0`
+
+- &#8203;<!-- 14 -->[styled-engine] Fix props inference in styled-engine (#29739) @Janpot
+
+### `@mui/base@5.0.0-alpha.56`
+
+- &#8203;<!-- 22 -->[FormControlUnstyled] `focused` is always false unless explicitly set to `true` @mwilkins91
+- &#8203;<!-- 12 -->[TabsUnstyled] Introduce new component (#29597) @mnajdova
+
```

Comment:
Reminder for the Codemod PR that is merged.

Reply:
I included it.